client/src/pages/Login/Login.tsx

```diff
@@ -24,16 +36,34 @@ export default function Login(): JSX.Element {
         }
       }
     } catch (err) {
+
+      // Log any errors to the console
       console.log(err);
     }
   };
 
+  // Render login form
   return (
-    <div className={styles['Login']}>
-      <p>Login Page</p>
-      <TextField header="Username or Email" setInputValue={setUserInput} />
-      <PasswordField header="Password" setInputValue={setPassword} />
-      <Button onClick={handleLogin}>Login</Button>
+    <div><p>Login Page</p>
```

Comment:
I would remove that Login Page paragraph, that was just filler so we knew what page we were on.

Reply:
Removed the login page paragraph. Thanks!